stories/LocationSelector.tsx

```diff
@@ -209,6 +209,7 @@ storiesOf('Organisms|LocationSelector', module)
                 radiusDefaultValue={number('Radius default value', 5)}
                 radiusStep={number('Radius steps', 1)}
                 placeTypes={['(regions)']}
+                shouldGetAddressInfo={boolean('Fetch address information', false)}
```

Comment:
Could you please add `shouldGetAddressInfo` for `Example single select implementation` as well?